fapi-client/src/main/scala/com/gu/facia/api/models/collection.scala

```diff
@@ -83,7 +98,21 @@ object Collection {
       collection.live
       .filter(_.isSnap)
       .filter(_.safeMeta.snapType == Some("latest"))
-      .flatMap(snap => snap.safeMeta.snapUri.map(uri => snap.id -> uri)).toMap)
+      .flatMap(snap => snap.safeMeta.snapUri.map(uri => snap.id -> uri))
+      .toMap)
+
+  def treatsRequestFor(collection: Collection): TreatsRequest = {
```

Comment:
I don't agree with in-lining this or with the `tuple` suggestion. This is purposely returning a `TreatsRequest` to deal with to two different methods of requesting items. Where were you suggesting to in-line this?